bin/git-cp

```diff
@@ -38,6 +38,18 @@ else
         exit 40
     fi
 
+    if [[ "$DESTINATION_FILENAME" == */ ]]; then
+      echo 1>&2 "$DESTINATION_FILENAME is not a file path."
+      exit 80
+    fi
+    DESTINATION_DIR="${DESTINATION_FILENAME%/*}"
+    if [ ! -d "$DESTINATION_DIR" ]; then
```

Comment:
We can use `mkdir -p` directly as the cmd won't return error if the dir exists?

Reply:
Make sense. Removed this check.